troi/patches/daily_jams.py

```diff
@@ -19,20 +19,21 @@ class DailyJamsElement(Element):
         Split weekly recommended recordings into 7 sets, one for each day of the week.
     '''
 
-    def __init__(self, recs, day):
+    def __init__(self, recs, user, day):
```

Comment:
we haven't been focusing on docstrings for `__init__` in these elements, but I think it's becoming more important and we should start adding brief documentation here